InvoiceManagementStudio/Model/Domain/InvoicePayment.cs

```diff
@@ -1,14 +1,107 @@
 using System;
```

Comment:
There are some unused imports provided for no reason.
Do those ship with VS? Does it not mark them as unused in your ide?

List of `used` imports, the rest should be deleted

- using System;
- using System.Globalization;
- using InvoiceManagementStudio.Core.Definition;
- using InvoiceManagementStudio.Core.Definition.Payment;